cns/middlewares/k8sSwiftV2.go

```diff
@@ -166,119 +165,10 @@ func (m *K8sSWIFTv2Middleware) getIPConfig(ctx context.Context, podInfo cns.PodI
 			PrefixLength: uint8(prefixSize),
 		},
 		MacAddress:        mtpnc.Status.MacAddress,
-		NICType:           cns.DelegatedVMNIC,
+		NICType:           mtpnc.Status.InterfaceInfos[0].NICType,
```

Comment:
check if length of this array is > 0, if so then read all nic-types from list - need to discuss with paul & shreyas on tying this behavior to cni->cns

Reply:
Does this include the PrimaryIP -> IPAddress in PodIpInfo?

Reply:
yes

Reply:
Addressed.